docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md

```diff
@@ -0,0 +1,125 @@
+---
+id: multi-namespace-deployment
+title: "Multi-namespace deployment"
+description: "Deploy Camunda 8 Self-Managed across namespaces"
```

Comment:
I would add a purpose here, so "Deploy Camunda 8 Self-Managed across namespaces so you can..."

Essentially, I'm just wondering what the benefit is so users may feel more inclined to click into the documentation.

Reply:
I've added more details in the description :+1: